extensions/amp-story/1.0/_locales/default.js

```diff
@@ -21,6 +21,15 @@ import {LocalizedStringBundleDef, LocalizedStringId} from '../localization';
  * @const {!LocalizedStringBundleDef}
  */
 export default {
+  [LocalizedStringId.AMP_STORY_AUDIO_MUTE_BUTTON_TEXT]: {
+    string: 'Sound off',
+  },
+  [LocalizedStringId.AMP_STORY_AUDIO_UNMUTE_SOUND_TEXT]: {
+    string: 'Sound on',
+  },
+  [LocalizedStringId.AMP_STORY_AUDIO_UNMUTE_NO_SOUND_TEXT]: {
+    string: 'This page has no sound',
```

Comment:
Can you test what happens if the text overflows?  What if this string (in some language) is really really long?  Does it break the UI?

Reply:
The UI still works (as in nothing breaks and the intended behavior works of the message showing out appropriately), but if the line is very long, it won't fit in the viewport so some of it gets cut off (even though the message animation works). Is this ok, or is there a workaround for very long strings?